### DISCUSSION TITLE
Eligibility for Mindsdb swag credits after PR merge

### DISCUSSION THREAD

**Discussion by kunaljubce** [mindsdb/mindsdb#6541] [General]
Hi Team - I made a doc update PR which got merged recently. This was my first contribution - https://github.com/mindsdb/mindsdb/pull/6517

Do I qualify for mindsdb swag credits for this submission?

![image](https://github.com/mindsdb/mindsdb/assets/34453918/22f9937e-3222-45bc-9f1f-e085ce01dd87)

**Comment by chandrevdw31** [comment]
Hey @kunaljubce yes you can claim here https://mindsdb.com/rewards
